### PR TITLE
Header basic version

### PR DESCRIPTION
**Pull request by umbobabo**
There are couple of temporary hack related to the fact that font is not still avaible so we can use @font-face as the final implementation will require...

>   font-family: 'Halifax Regular';

... and come some styles are temporary due to the missing of the final illustration...

> .header__title {
>  text-shadow: 2px -2px var(--header__title-text-shadow, var(--color-chicago)),
>  -2px 2px var(--header__title-text-shadow, var(--color-chicago)),
>  -2px -2px var(--header__title-text-shadow, var(--color-chicago)),
>  2px 2px var(--header__title-text-shadow, var(--color-chicago));
> }
...just to keep text readable also if the illustration have the same color of the text.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
+@import '@economist/component-typography';
+@import '@economist/component-palette';
+@import '@economist/component-grid';
+
+@custom-media --viewport-big (width >= 36rem);
+@custom-media --viewport-small (width <= 35.9999rem);
```
Comment (fabiosantoscode) [inline index.css]: You can just say `width < 36em`, right?

File: index.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    margin-top: 0;
+  }
+
+  .header__group-text,
+  .header__group-image {
+      flex: 1;
+  }
+
+  .header__group-image {
+      position: relative;
```
Comment (fabiosantoscode) [inline index.css]: inconsistent indentation here

File: index.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    width: 100%;  /* Fix for IE */
+  }
+
+  .header__title {
+    margin-top: 0;
+  }
+
+  .header__group-text,
+  .header__group-image {
+      flex: 1;
```
Comment (fabiosantoscode) [inline index.css]: See below :arrow_down:

**Comment by fabiosantoscode** [comment]
Left some comments but all optional. Look at them, and if you're happy with it the way it is, :rainbow:.

File: index.css (hunk 1 above)
Comment (umbobabo) [inline index.css]: Unfortunately not.
Probably is a css-next issue but If I write like that the media query is not transpiled.
The "=" seems to be required.

**Pull request merged**